src/main/java/uk/gov/hmcts/reform/prl/controllers/CallbackController.java

```diff
@@ -494,7 +494,7 @@ public AboutToStartOrSubmitCallbackResponse aboutToSubmitCaseCreation(
         }
 
         // Saving the logged-in Solicitor and Org details for the docs..
-        caseDataUpdated = getSolicitorDetails(authorisation, caseDataUpdated);
+        getSolicitorDetails(authorisation, caseDataUpdated);
```

Comment:
@pete-lowe , do you think this can cause any issue? we have fixed something like this for court name not getting populated issue, if you remember?

Reply:
changes reverted based on the discussion and previous bug identification.